webplugin/template/mck-sidebox.html

```diff
@@ -3,10 +3,10 @@
 		<div id="mck-sidebox-content" class="mck-sidebox-content mck-box-content">
 			<div id="mck-conversation-header" class="mck-conversation-header mck-truncate n-vis"></div>
 			<div class="mck-box-top mck-title">
-				<div class="blk-lg-10">
+				<div class="mck-header-container">
```

Comment:
@akshatdhabi  Please use "Km" instead of "mck" here.
We were using "mck" declarations for Applozic and Kommunicate we are using "km".